docs/guides/cloud/free-trial.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: Free Trial
+sidebar_position: 15
+---
+
+:::info
+
+## <Icon name="graduation-cap" /> What you'll learn
+
+- What is available during your free trial of Cypress Cloud
+- What it means to connect Cypress Cloud to your Cypress tests
+- How to use the onboarding guide included in the free trial
+
+:::
+
+## Structure of the Cypress Cloud free trial
+
+Your free trial of Cypress Cloud provides a 14 day all-access pass to Cypress Cloud features.
+This is the same list of Cloud features available in our Enterprise plan as described on the
+[Cypress Cloud pricing page](https://www.cypress.io/pricing). 50,000 test results are included!
+
+Experiencing the capabilities of Cypress Cloud during a trial is the quickest way to realize
+that your testing life wouldn't be the same without it! After starting your trial you can select
+and subscribe to a Cypress Cloud plan at any time.
+
+However, if you determine that subscribing to a Cloud plan is
+not workable at the end of 14 days, rest assured that your account will remain available to you
+and simply revert to our _Starter_ plan. The _Starter_ plan is free, but does not have the same
+set of features. The _Starter_ plan will also limit data retention to 30 days and test results
+to 500 per monthly period. Details of the _Starter_ plan are available on the
+[pricing page](https://www.cypress.io/pricing).
+
+:::caution
+
+<Icon name="exclamation-triangle" /> The only restriction during a free trial is
+that you are not allowed to [transfer](/guides/cloud/account-management/projects#Transfer-ownership)
+existing Cypress Cloud projects into your free trial organization.{' '}
```

Comment:
Is this necessary at the end of the line?

`{' '}`

Reply:
no.  :)